src/handlers/utils.rs

```diff
@@ -12,6 +12,8 @@ pub fn health_handler() -> Json<HealthResponse> {
     let status = String::from("ok");
     let health_response = HealthResponse {status};
 
+    info!("Health handler executed successfully");
```

Comment:
Can we change this to be `debug`? I think these are less important, and only relevant when debugging